src/main/java/com/habitmon/domain/habitmon/domain/HabitMon.java

```diff
@@ -15,6 +15,9 @@ public class HabitMon {
     @Id
     private Long id;
 
+    @NotNull
+    private Long memberId;
```

Comment:
연관관계 매핑해주시면 좋을 것 같아요~!!
```suggestion
    @NotNull
    @OneToOne
    private Member member;
```